app/build.gradle

```diff
@@ -77,6 +77,23 @@ def cordaCLi = tasks.named('jar', Jar) {
     archiveVersion = project.version
 }
 
+tasks.register('publishBuildkitImage', BuildkitBuild) {
```

Comment:
for clarity to others that may stumble upon this task please add a comment here that this is a alternative to JIB we are trialing  it will be rolled out first to the nighties along with select local testing.
Once stability is confirmed we will is a phased manner look to phase out original task